src/components/Contentful/HomePage/LoansByCategoryGrid.vue

```diff
@@ -0,0 +1,79 @@
+<template>
```

Comment:
Why do we need a nearly exact copy of the existing component?
The only thing different from https://github.com/kiva/ui/blob/main/src/components/Contentful/LoansByCategoryCarousel.vue is the name property...doesn't seem necessary.

Reply:
It also appears to not be used...